modules/shared.py

```diff
@@ -115,6 +115,7 @@ def str2bool(v):
 parser.add_argument('--model_type', type=str, help='GPTQ: Model type of pre-quantized model. Currently LLaMA, OPT, and GPT-J are supported.')
 parser.add_argument('--groupsize', type=int, default=-1, help='GPTQ: Group size.')
 parser.add_argument('--pre_layer', type=int, default=0, help='GPTQ: The number of layers to allocate to the GPU. Setting this parameter enables CPU offloading for 4-bit models.')
+parser.add_argument('--warmup_autotune', action=argparse.BooleanOptionalAction, default=True, help='GPTQ: Enable warmup autotune. Only usable for triton.')
```

Comment:
Wouldn't it be better to set this to false by default? Otherwise the only use of this parameter is to define `--no-warmup_autotune` which is not obvious

Reply:
@oobabooga in triton branch it defaults to True, and it (kind of) fixes the startup delay when generating, but introduce ~30s delay during loading.
Should I just make this `--no-warmup_autotune` and defaults to False(default autotune on)?

Reply:
It takes extra time in loading but instead makes the model run faster. Because this set default true.